pontoon/base/models.py

```diff
@@ -3021,11 +3014,6 @@ def map_entities(
                     "obsolete": entity.obsolete,
                     "translation": translation_array,
                     "readonly": entity.resource.project.projectlocale[0].readonly,
-                    "visible": (
-                        False
-                        if entity.pk not in visible_entities or not visible_entities
-                        else True
-                    ),
```

Comment:
Took me a while to investigate where this comes from - good old (deprecated) in-context l10n!